src/main/resources/sql/court/insert-court.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO court (court_name,phone,extension,website,payment_system,address,city,state,zip_code,latitude,longitude,citation_expires_after_days) VALUES (:name,:phone,:extension,:website,:paymentSystem,:address,:city,:state,:zipCode,:latitude,:longitude,:citationExpires)
```

Comment:
You shouldn't need this file. Spring Data should eliminate the need for this. Plus, nothing references this file anyway.

Reply:
Yes, this code was left over from pre-spring data conversion.